{"uuid":"e58527eb-5214-4313-8a2b-78d1e2d7e713","historyId":"dca78036dabddc8ee02c1f0c01951305","fullName":"PutUserInformationTest.putUserNegativeTest","labels":[{"name":"package","value":"PutUserInformationTest"},{"name":"testClass","value":"PutUserInformationTest"},{"name":"testMethod","value":"putUserNegativeTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"ntcn"},{"name":"subSuite","value":"PutUserInformationTest"},{"name":"host","value":"Mac-mini-Belikova.local"},{"name":"thread","value":"4019@Mac-mini-Belikova.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"PUT /v1/user"}],"links":[],"name":"putUserNegativeTest","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\nThe following assertion failed:\n1) \nExpecting:\n <1>\nto be equal to:\n <7>\nbut was not.\nat PutUserInformationTest.putUserNegativeTest(PutUserInformationTest.java:109)\n","trace":"org.assertj.core.api.SoftAssertionError: \nThe following assertion failed:\n1) \nExpecting:\n <1>\nto be equal to:\n <7>\nbut was not.\nat PutUserInformationTest.putUserNegativeTest(PutUserInformationTest.java:109)\n\n\tat org.assertj.core.error.AssertionErrorCreator.multipleSoftAssertionsError(AssertionErrorCreator.java:75)\n\tat org.assertj.core.api.AbstractSoftAssertions.assertAll(AbstractSoftAssertions.java:38)\n\tat org.assertj.core.api.AbstractSoftAssertions.assertAll(AbstractSoftAssertions.java:43)\n\tat PutUserInformationTest.putUserNegativeTest(PutUserInformationTest.java:112)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:766)\n\tat org.testng.TestRunner.run(TestRunner.java:587)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1109)\n\tat org.testng.TestNG.runSuites(TestNG.java:1039)\n\tat org.testng.TestNG.run(TestNG.java:1007)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","description":"Negative test","steps":[],"attachments":[],"parameters":[],"start":1656328168606,"stop":1656328173106}